lib/auto-languageclient.js

```diff
@@ -438,4 +443,13 @@ export default class AutoLanguageClient {
       this._signatureHelpRegistry = null;
     });
   }
+
+  /**
+   * `didChangeWatchedFiles` message filtering, override for custom logic.
+   * @param filePath path of a file that has changed in the project path
+   * @return false => message will not be sent to the language server
+   */
+  filterChangeWatchedFiles(filePath: string): boolean {
+    return true;
```

Comment:
If we make this filterChangeWatchedFiles method take the filePath and the projectPath we can move the default here while also providing more flexibility.